### PR TITLE
Project cards

### PR DESCRIPTION
**Pull request by kimcharlene**
YAAY!

This pull request introduces new major features:
- Project preview card components (with card heading, description, and images fetched with queries and individual .mdx files)
- Added associated project images and filler text for the Design section
- SEO.js for better tab naming

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,176 +1,45 @@
...
-const IndexPage = () => {
+// Step 1: Import React
+import * as React from 'react'
+import { graphql } from 'gatsby'
+import Layout from '../components/layout'
+import ProjectCard from '../components/project-card.js'
+import Seo from '../components/seo'
+
+// Step 2: Define your component
+const TeachingPage = ({ data }) => {
```
Comment (seongsikk) [inline src/pages/index.js]: It seems like you are defining TeachingPage here as well as "src/pages/teaching.js'? Did you mean to do this?

**Review by seongsikk — COMMENTED** [review]
Can you also please add some screenshots to the description of the pages that you created so far for all of the PRs? Thanks!

**Comment by kimcharlene** [comment]
pr#2- fig1
Screenshot of project info cards highlighted.

**Comment by seongsikk** [comment]
Looks good!

**Pull request merged**